### PR TITLE
[VIP-2622] Chaos labs recommendations (May 29th, 2024)

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2622

### CODE REVIEW

File: vips/vip-315/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const vWBNB_IR = "0xdE41feaB3B17C05Ba596b11E2C8d9f3514B71d22";
+export const vWBNB = "0xe10E80B7FD3a29fE46E16C30CC8F4dd938B742e2";
+export const vWBNB_RF = parseUnits("0.5", 18);
```
Comment (chechu) [inline vips/vip-315/bscmainnet.ts:8]:
```suggestion
export const vWBNB_RF = parseUnits("0.05", 18);
```

File: vips/vip-315/bscmainnet.ts [hunk 2]
```diff
@@ -56,6 +58,16 @@ const vip315 = () => {
...
+        params: [[vstkBNB], [stkBNB_SUPPLY_CAP]],
+      },
+      {
+        target: COMPTROLLER,
+        signature: "setMarketBorrowCaps(address[],uint256[])",
+        params: [[vstkBNB], [stkBNB_BORROW_CAP]],
+      },
     ],
     meta,
     ProposalType.REGULAR,
```
Comment (chechu) [inline vips/vip-315/bscmainnet.ts:73]: I would make FAST_TRACK

File: simulations/vip-315/bscmainnet.ts [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        [4, 4, 1, 1],
+      );
+    },
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("check IR address", async () => {
+      const ir = await vWBNBContract.interestRateModel();
+      expect(ir).to.be.equal(vWBNB_IR);
+    });
```
Comment (chechu) [inline simulations/vip-315/bscmainnet.ts:103]: We could also use the `checkInterestRate` function, as we did at [VIP-308](https://github.com/VenusProtocol/vips/blob/main/simulations/vip-308/bscmainnet.ts#L7)

**Review by chechu — APPROVED** [review]

**Review by kkirka — APPROVED** [review]

**Review by Debugger022 — APPROVED** [review]
LGTM :)

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/318

**Pull request merged**